package.json

```diff
@@ -20,6 +20,7 @@
     "start": "react-app-rewired start",
     "prestart": "yarn run version-file",
     "build": "react-app-rewired build",
+    "build:readme": "./node_modules/.bin/toctoc -w -d 2 README.md",
```

Comment:
I guess we need to add toctoc to the devDependencies, also in the package.json you can remove `./node_modules/.bin` and use `toctoc` directly